app/data/models/vet-visit.js

```diff
@@ -0,0 +1,35 @@
+const createReference = require('../../lib/create-reference')
+module.exports = (sequelize, DataTypes) => {
+  return sequelize.define('vetVisit', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      autoIncrement: true,
+      defaultValue: sequelize.UUIDV4
+    },
+    reference: {
+      type: DataTypes.STRING,
+      set (val) {
+        this.setDataValue('reference', val.toUpperCase())
+      }
+    },
+    applicationReference: DataTypes.STRING,
+    rcvs: DataTypes.STRING,
```

Comment:
For RCVS, I would use the same pattern as in the application table where non-searchable fields and stored in the `data` column, unless there is a reason for `rcvs` to be its own column?